index.html

```diff
@@ -1,46 +1,69 @@
-    </div>
-    <!-- div class="video-suggestions">
-      <ul>
-        <li ><iframe width="1280" height="720" src="https://www.youtube.com/embed/kl4fLB9-V8A" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></li>
-        <li><iframe width="1280" height="720" src="https://www.youtube.com/embed/k2qgadSvNyU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></li>
-        <li><iframe width="1280" height="720" src="https://www.youtube.com/embed/oiKj0Z_Xnjc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></li>
-      </ul>
-    </div>-->
-    <aside class="sidebar-suggestions">
-      <div class="videos-suggestion">
-        <div class="thumbnail-video">
-          <iframe width="1206" height="678" src="https://www.youtube.com/embed/Ks3_kuRAzHs" frameborder="0"
-            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-        </div>
-        <div class="information">
-          <h3>Nicky Minaj: Whatever</h3>
-          <p>Views: 7,456,452.</p>
-        </div>
-      </div>
-    </aside>
-  </section>
-
+  <nav class="gradient-header">
+    <ul class="global-nav">
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+      <li class="nav-items">
+        <a href=""><span>Apple</span></a>
+      </li>
+    </ul>
+  </nav>
+  
+  <article class="billboard">
+    <section class="container dashboard">
+      <header>
+        <h1 class="title">What will your verse be?</h1>
+        <h2 class="sub-headline">Everyone has something to share. See how people are using iPad in their unique ways <br/> to add to the world’s story.</h2>
+        <h2 class="title">What will your verse be?</h2>
+      </header>
+    </section>
+  </article>
+  <aside class="promos">
+    <ul>
+      <li>
+        <img class="promo-img" src="./assets/amALion.jpg" alt="">
+      </li>
+      <li>
+        <img class="promo-img" src="./assets/amALion.jpg" alt="">
+      </li>
+      <li>
+        <img class="promo-img" src="./assets/amALion.jpg" alt="">
+      </li>
+      <li>
+        <img class="promo-img" src="./assets/amALion.jpg" alt="">
```

Comment:
Is there a reason why we using a different image from the source page? :thinking: